api/src/main/resources/IdentifierSource.hbm.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0"?>
 <!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd" >
-
 <hibernate-mapping package="org.openmrs.module.idgen" >
 
-	<class name="IdentifierSource" table="idgen_identifier_source" abstract="true">
-	
-		<id name="id" type="int" column="id" unsaved-value="0"><generator class="native" /></id>
+	<class name="IdentifierSource" table="idgen_identifier_source" abstract="true" >
+		<id name="id" type="int" column="id" unsaved-value="0">
+			<generator class="native" />
+		</id>
+		<discriminator column="source_type" />
```

Comment:
Surely there will be a need for a Liquibase changeset that sets the discriminator values in the new **source_type** column for an existing **idgen_identifier_source** table?